examples/fused_linear_jsd.py

```diff
@@ -0,0 +1,122 @@
+from __future__ import annotations
```

Comment:
please add test to test_examples.py too, following other examples in this folder